### ISSUE TITLE
ManagedCuda.NPP.NPPException:“Device allocation error”

### ISSUE THREAD

**Issue by sheuchenko** [kunzmi/ImageStackAlignator#24]
Hi Michael, Thanks for the helpful job.
I meet an error and have not got the final results. I have finished step1 and 2 and 3 successfully, but when clicking **Prepare accumulation**, it shows the exception: ManagedCuda.NPP.NPPException:“Device allocation error” .
Could you please help me to settle this problem?
Thank you so much~

**Comment by kunzmi** [comment]
Hi, can you provide a bit more information, like size and number of images? Super-resolution mode? What GPU? But after all, if NPP, or cuda, complains about ```Device allocation error```, this nearly always means that you run out of memory on GPU

**Comment by sheuchenko** [comment]
Yes! It runs out of memory on GPU because I use the large dng captured by Huawei P30(each photo is 78M). Now I use the photos with lower resolution, it runs successfully. Thank you so much!

**Comment by iKaHibi** [comment]
> Yes! It runs out of memory on GPU because I use the large dng captured by Huawei P30(each photo is 78M). Now I use the photos with lower resolution, it runs successfully. Thank you so much!

Hi sheuchenko! I am wondering how can I get synthetic image bursts by Huawei P30. Can you tell me how did you do that?
